PPOCRLabel.py

```diff
@@ -3665,6 +3679,89 @@ def expandSelectedShape(self):
             self.updateBoxlist()
             self.setDirty()
 
+    def resortBoxposion(self):
```

Comment:
Mis-spelled method name

```suggestion
    def resortBoxPosition(self):
```